src/Compilers/CSharp/Portable/Binder/Binder_Expressions.cs

```diff
@@ -7560,7 +7543,7 @@ private BoundExpression ResolveToExtensionMemberIfPossible(BoundExpression expr,
             }
         }
 
-        private BoundExpression BindInstanceMemberAccess(
+        private BoundExpression BindMemberAccessWithBoundLeftInternal(
```

Comment:
>Internal [](http://example.com/codeflow?start=61&length=8)

What are we trying to convey with the "Internal" suffix? It doesn't look meaningful to me. Consider dropping it. If your goal is as simple as to not share name with the other helper, numeric suffixes "BindMemberAccessWithBoundLeft1" and "BindMemberAccessWithBoundLeft2" will be better in my opinion.

Reply:
Yes, my main goal is to remove the `Instance` portion (misleading) and not share names since we have a cref reference to it. How about "Core" suffix?

Reply:
> How about "Core" suffix?

I do not find this better.  A numeric suffix is better in my opinion to avoid sharing the name. Unless we can come up with a meaningful differentiator.

Reply:
Since you already changed the name, I guess I can live with it.